testsuite/cluster-test/src/suite.rs

```diff
@@ -35,16 +35,16 @@ impl ExperimentSuite {
             experiments.push(b);
         }
         experiments.push(Box::new(
-            PerformanceBenchmarkParams::new_nodes_down(0).build(cluster),
+            PerformanceBenchmarkThreeRegionSimulationParams {}.build(cluster),
```

Comment:
Can we actually invest time and fix underlining problem, e.g. add healthcheck for full node?
Does not have to be fancy, some prometheus query checking latest version will suffice. I am concerned that we are just sweeping some real problem that we already seen affecting us under the rug

Reply:
I was hoping you can help with adding healthcheck for full node. Once we have that, we can switch our CI to switch to the validator + fullnode setup.

Reply:
I have to focus on smth else for at least next few days(and possibly next week). I would rather prefer not to shuffle experiment results in output, as people read them and it will create extra confusion.
Do you think this is something you could look at? (alternatively I guess we can just wait with this migration until I get to that, but thats less preferred ( )

Reply:
Re-ordered the experiments. Can we proceed with landing this PR?